ci/terraform/oidc/staging.tfvars

```diff
@@ -8,3 +8,5 @@ logging_endpoint_arns                = []
 shared_state_bucket                  = "di-auth-staging-tfstate"
 test_clients_enabled                 = true
 internal_sector_uri                  = "https://identity.staging.account.gov.uk"
+
+orch_client_id = "orchestrator"
```

Comment:
This should be `orchestrationAuth` and is already declared in `staging-overrides.tfvars`.